<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersExample</class>
 <widget class="QWidget" name="ParametersExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>This is a parameter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="param2Combo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Covariance Matrix type</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Method 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Method 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Method 3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>78</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>This is another one</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="param3Check">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Do Something</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="param1Spin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
